docs/zephyr_01_high_level_requirements.sdoc

```diff
@@ -43,10 +43,10 @@ TYPE: High Level
 COMPONENT: Hardware Architecture Interface
 TITLE: Architecture Layer Interface
 STATEMENT: >>>
-Zephyr shall provide a framework to communicate with a set of hardware architectural services.
+The Zephyr RTOS shall provide a framework to communicate with a set of hardware architectural services.
```

Comment:
Let's isolate these `The Zephyr RTOS` changes into a separate PR because these are trivial changes that could be reviewed and merged independently of the Mutex* review which is clearly more advanced.

Reply:
Are we switching name from OS to RTOS?

https://docs.zephyrproject.org/latest/ still says "Zephyr OS"

I saw it as "product" name.
Whether OS or RTOS I do not care so much. It it has been decided to become RTOS over OS it would be nice if consistent.

Hence if changing it here, please also in the doc and the code.
Therefore as Stan said -> own MR make more sense.